UPGRADE-3.x.md

```diff
@@ -1,6 +1,14 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.25 to 3.x
+========================
+
+### SonataEasyExtends is deprecated
+
+Registering `SonataEasyExtendsBundle` bundle is now deprecated, you should not register it,
+and instead register the `SonataDoctrineBundle` bundle.
```

Comment:
IMO, the term "now" is pointless in a file intended to keep continuous track of the history.
```suggestion
Registering `SonataEasyExtendsBundle` bundle is deprecated, it SHOULD NOT be registered.
Register `SonataDoctrineBundle` bundle instead.
```